java/src/test/java/vowpalWabbit/learner/VWActionScoresLearnerTest.java

```diff
@@ -4,16 +4,65 @@
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 import vowpalWabbit.VWTestHelper;
+import vowpalWabbit.responses.ActionScore;
 import vowpalWabbit.responses.ActionScores;
 
 import java.io.IOException;
 
 import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertTrue;
 
 /**
  * Created by jmorra on 10/2/15.
  */
 public class VWActionScoresLearnerTest extends VWTestHelper {
+    private final float TOLERANCE = 0.001f;
```

Comment:
None of this is necessary as it is already in `Assert`.

Reply:
See comment below about using the `Object` overload meaning tolerance cannot be used